src/edge_kernels/MomentumEdgeSolverAlg.C

```diff
@@ -126,6 +135,8 @@ MomentumEdgeSolverAlg::execute()
 
       const DblType pecfac = pecletFactor.get(edge, 0);
       const DblType om_pecfac = 1.0 - pecfac;
+      maxPeclet.get(nodeR, 0) = stk::math::max(maxPeclet.get(nodeR, 0), pecfac);
+      maxPeclet.get(nodeL, 0) = stk::math::max(maxPeclet.get(nodeL, 0), pecfac);
```

Comment:
This most likely won't do what you expect it to do on GPUs.

Reply:
:+1: will refactor